### PR TITLE
Add schnitzel page. Add connection to schniztel page.

### PR DESCRIPTION
**Pull request by JaydenMatas**
Please review this, @casey-collab

### CODE REVIEW

**Comment by JaydenMatas** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Hey! Me! Review this @casey-collab

If it cannot be merged just mention me and I'll check it again.

File: english/run_out_the_door/run_out_the_door.md [hunk 1]
```diff
@@ -24,3 +24,5 @@ You...
 Decide to [start dancing bachata](../start-dancing/startDancing.md) and everybody start to clap and cheer you up.
 
 [Trip and stumble into a pitch-black cave.](grue/grue.md)
+
+Forget about everything and go and find a good pub feed such as a [Chicken schniztel](../schniztel/schnitzel.md)
```
Comment (casey-collab) [inline english/run_out_the_door/run_out_the_door.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/schnitzel/schnitzel.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+The Schniztel goes down well. But now you feel like having
```
Comment (casey-collab) [inline english/schnitzel/schnitzel.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**